data/hai/hai reveal 4 middle.txt

```diff
@@ -181,7 +181,7 @@ mission "Hai Reveal [A03a] Symbol Identification"
 		log `Been asked to check out the Hai spaceports in various systems near the wormhole, just to check nothing unexpected is happening.`
 		conversation
 			`You make what is beginning to feel like a standard request to land at the secure spaceport and disembark with relevant copies of files on a data stick in hand. It doesn't take long to track down Alondo, though you do end up having to wait for nearly half an hour while he finishes up some kind of meeting with a Hai official of some kind. When he's finished he waves you into the room gestures for you to take a seat, though the seat in question is clearly designed to be ergonomic for a Hai which makes for a rather strange experience for a human.`
-			`	<You have a conversation where you show him the symbol, and ask him for leads. He says he knows some people who can be discreet and he'll see if he can find someone who might know something. Also something about him working with Remington lately.>`
+			`	You have a conversation where you show him the symbol, and ask him for leads. He says he knows some people who can be discreet and he'll see if he can find someone who might know something. Also something about him working with Remington lately.`
```

Comment:
This is actually kind of intentional, this line is on the to-do list to actually have that conversation written up.

Reply:
I want this to be a bugfix-only PR, so I won't add actual content here. Instead, I'll just make the filler less glaring and add a "TODO" comment above it.